package.json

```diff
@@ -21,10 +21,13 @@
         },
         {
             "name": "James Thomas"
+        },
+        {
+            "name": "Raymond Mouthaan"
         }
     ],
     "dependencies": {
-        "node-red": "0.18.4",
+        "node-red": "0.20.5",
         "node-red-node-msgpack": "*",
```

Comment:
Extra nodes not really needed - were here to sort of match the Pi pre-install - but are not core nodes.
There is also an engines statement in this file that should be updated/fixed to at least 8.x

Reply:
The package.json in the new coming PR looks like this:

```
...
    "dependencies": {
        "node-red": "0.20.7"
    },
    "engines": {
        "node": "8.*.*"
    }
...
```

Would that be sufficient?